internal/cli/atlas/backup/compliancepolicy/copyprotection.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compliancepolicy
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201003/admin"
+)
+
+type CopyProtectionOpts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	policy *atlasv2.DataProtectionSettings
+	store  store.CompliancePolicy
+	enable bool
+}
+
+const (
+	enable  string = "enable"
+	disable string = "disable"
+	active  string = "ACTIVE"
```

Comment:
type should redundant on initialization 
```suggestion
	enable  = "enable"
	disable = "disable"
	active  = "ACTIVE"
```

Reply:
Could you elaborate a bit more on this? I'm not disagreeing with you, just want to use this as a learning opportunity. My reasoning was that without `string`, the type is `untyped string` (`const enable untyped string = "enable"`). With `string`, it's what you'd expect of course (`const disable string = "disable"
`). Why is this worse?

Reply:
Thanks for the clarification, change implemented.